greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExport.java

```diff
@@ -0,0 +1,139 @@
+package com.greenback.kit.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class AutoExport extends GreenbackObject{
+    private String userId;
+    private String accountingAccountId;
+    private AutoExportRequest parameters;
+    private AutoExportFrequency frequency;
+    private AutoExportState state;
+
+    // from expands
+    private Account accountingAccount;
+    private List<Account> accounts;
+    private AutoExportRun lastRun;
+    private List<AutoExportRun> runs;
+
+    public String getUserId() {
+        return userId;
+    }
+
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    public String getAccountingAccountId() {
+        return accountingAccountId;
+    }
+
+    public void setAccountingAccountId(String accountingAccountId) {
+        this.accountingAccountId = accountingAccountId;
+    }
+
+    public AutoExportRequest getParameters() {
+        return parameters;
+    }
+
+    public void setParameters(AutoExportRequest parameters) {
+        this.parameters = parameters;
+    }
+
+    public AutoExportFrequency getFrequency() {
+        return frequency;
+    }
+
+    public void setFrequency(AutoExportFrequency frequency) {
+        this.frequency = frequency;
+    }
+
+    public AutoExportState getState() {
+        return state;
+    }
+
+    public void setState(AutoExportState state) {
+        this.state = state;
+    }
+
+    public AutoExportRun getLastRun() {
+        return lastRun;
+    }
+
+    public void setLastRun(AutoExportRun lastRun) {
+        this.lastRun = lastRun;
+    }
+
+    public List<AutoExportRun> getRuns() {
+        return runs;
+    }
+
+    public void setAllCompletedRuns(List<AutoExportRun> runs) {
```

Comment:
You'll need to compute these somewhere else, maybe a utility class of sorts which accepts a List<AutoExportRun> runs?